### DISCUSSION TITLE
`Field list` and `Definition list` give same output?

### DISCUSSION THREAD

**Discussion by gunungpw** [pradyunsg/furo#422] [Potential Issue] [answered]
Hello, thank you for maintainer and contributor of Furo Theme:
I don't know if this is an issue or expected result but from `docutils` and `http://rst.aaroniles.net/`. Definition list and field list will give different output but in furo it will give the same output:

Furo version:
`2022.4.7`
Sphinx version:
`4.5.0`
Docutils version:
`0.17.1`

example `test.rst`:
```
Definiton list
--------------

Description
   bla bla bla

default
   ``True``

Field list
----------

:Description: bla bla bla

:Default: ``True``
```
output:
![image](https://user-images.githubusercontent.com/55311527/163676802-e8bd9471-34fa-406c-8378-39cd48b224d3.png)

expected output:

### Definition list
```
Description
    bla bla bla
default
    ``True``
```
### Field list
```
Description: bla bla bla
default: `True`
```

and in also show [docutils docs](https://docutils.sourceforge.io/docs/user/rst/quickref.html#definition-lists) definition list and field list will give different output:
![image](https://user-images.githubusercontent.com/55311527/163676637-a4fe7ab5-5b85-4752-bbbe-be8d7b2347f2.png)

Thank you :)

**Comment by pradyunsg** [comment]
Hmm... This seems to have slipped through the cracks.

**Comment by pradyunsg** [comment, marked as answer]
https://github.com/pradyunsg/furo/issues/719 filed for this.
